importer/api/__init__.py

```diff
@@ -112,6 +122,7 @@ def import_course(bundle, repo_id, user_id):
         user_id=user_id,
     )
     import_children(course, src, None)
+    return course
 
 
 def import_children(course, element, parent):
```

Comment:
I expected that parsing the static asset XML would be part of the import. Where in this pipeline did you plan for the parsing to take place?